specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/preview/2020-04-01-preview/changeanalysis.json

```diff
@@ -377,7 +396,12 @@
         "properties": {
           "$ref": "#/definitions/ConfigurationProfileResourceProperties"
         }
-      }
+      },
+      "location": "string",
+      "name": {
```

Comment:
name and id are also there already in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json